postgres/codenamemap.yaml

```diff
@@ -10,25 +11,26 @@
 
 {% macro debian_codename(name, version, codename=none) %}
 
-  {# use upstream version if configured #}
   {% if repo.use_upstream_repo == true %}
     {% set version = repo.version %}
     {% set fromrepo = repo.fromrepo|default(name ~ '-pgdg', true) %}
   {% else %}
     {% set fromrepo = name %}
   {% endif %}
+  {% set conf_dir = '/etc/postgresql/' + version|string + '/main' %}
+  {% set data_dir = '/var/lib/postgresql/' + version|string + '/main' %}
```

Comment:
That's OK, but generally better to use Jinja concatenation operator `~` (tilde) or `file.join` Salt's function here instead of type casting on variables.

Reply:
Forgot about tilde - very useful operand in this case. thx

> ~
Converts all operands into strings and concatenates them.